### ISSUE TITLE
Blacklist

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [TheCricket/Chisel-2#319]
We need some sort of blacklist.

If you have Erebus installed you can convert cobblestone back and forth. But erebus cobblestone is supposed to be somewhat rarer to acquire.

Same thing if you have Minefactory Reloaded. Black/White cobblestone and cobblestone conversion ignores the ingredient needed for black/white. Either make the colored cobblestone conversion + vanilla <-> erebus conversion cost way, way more durability on the tool or a blacklist would be nice. I prefer the first method, costing more durability.

**Comment by warlordjones** [comment]
More durability cost doesn't sound like a good solution.

I may be wrong but I"m fairly sure that the reason you can convert between black/white cobble & erebus cobble is because they are oredictionary registered as cobblestone - and therefore this isn't chisel's fault, as far as I know.

**Comment by Unrepentant-Atheist** [comment]
That's fine. That's were the blacklist would come in.

**Comment by warlordjones** [comment]
Although, the other mods shouldn't really be oredicting their cobble to normal cobble, if its so much rarer....

**Comment by Drullkus** [comment]
^

**Comment by tterrag1098** [comment]
Indeed, that is the root of the problem. The oredict is for EQUIVALENT items.
